src/TraceEvent/TraceEvent.csproj

```diff
@@ -54,7 +54,7 @@
   </Choose>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent.SupportFiles" Version="1.0.7" PrivateAssets="all"/>
+    <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent.SupportFiles" Version="1.0.7" />
```

Comment:
@vancem Can you describe the issue that led to this? I want to make sure I understand the intended distribution mechanism for these files.

Reply:
I am not certain what your question is, but the goal was to simply get simple scenarios that need OSExtensions (Starting a kernel session, or a heap session).